docs/data/sdks/browser@2/index.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Browser SDK 2.0
+description: The Amplitude Browser SDK Installation & Quick Start guide.
+icon: simple/javascript
+---
+
+
+![npm version](https://img.shields.io/npm/v/@amplitude/analytics-browser/next@2)
+
+The Browser SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-TypeScript).
+
+!!!info "Browser SDK Resources"
+    [:material-github: GitHub](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-TypeScript/releases)
+
+--8<-- "includes/ampli-vs-amplitude.md"
+    Click here for more documentation on [Ampli for Browser](./ampli.md).
+
+## Getting started
+
+Use [this quickstart guide](../sdk-quickstart#browser) to get started with Amplitude Browser SDK.
+
+## Usage
+
+### Initialize the SDK
+
+--8<-- "includes/sdk-httpv2-notice.md"
+
+--8<-- "includes/sdk-ts-browser/init.md"
+
+### Configuration
+
+--8<-- "includes/sdk-ts/shared-ts-configuration.md"
+    |`appVersion` | `string`. Sets an app version for events tracked. This can be the version of your application. For example: "1.0.0" | `undefined` |
+    |`defaultTracking` | `boolean | DefaultTrackingOptions`. Configures default event tracking | Check [tracking default events](./#tracking-default-events)|
+    |`deviceId` | `string`. Sets an identifier for the device running your application. | `UUID()` |
+    |`cookieOptions.domain` | `string`. Sets the domain property of cookies created. | `undefined` |
+    |`cookieOptions.expiration` | `number`. Sets expiration of cookies created in days. | 365 days |
+    |`cookieOptions.sameSite` | `string`. Sets `SameSite` property of cookies created. | `Lax` |
+    |`cookieOptions.secure` | `boolean`. Sets `Secure` property of cookies created. | `false` |
+    |`identityStorage` | `string`. Sets storage API for user identity. Options include `cookie` for `document.cookie`, `localStorage` for `localStorage`, or `none` to opt-out of persisting user identity. | `cookie` |
+    |`partnerId` | `string`. Sets partner ID. Amplitude requires the customer who built an event ingestion integration to add the partner identifier to `partner_id`. | `undefined` |
+    |`sessionTimeout` | `number`. Sets the period of inactivity from the last tracked event before a session expires in milliseconds. | 1,800,000 milliseconds (30 minutes) |
+    |`storageProvider`| `Storage<Event[]>`. Sets a custom implementation of `Storage<Event[]>` to persist unsent events. | `LocalStorage` |
```

Comment:
The default should be `CookieStorage` or `MemoryStorage`?

Reply:
default is local storage